xonsh/history.py

```diff
@@ -432,26 +621,30 @@ def _gc(ns, hist):
 
 
 _MAIN_ACTIONS = {
-    'show': _show,
+    'session': History.show,
```

Comment:
Maybe session should just be renamed show, or one should be an alias for the other.